book/website/community-handbook/local-build.md

```diff
@@ -0,0 +1,79 @@
+6. Change into the repo: `cd the-turing-way`
+7. Then change into the sub-directory the website is built from using `cd book/website`
+8. The Turing Way Book is built using multiple python libraries. We can install these dependencies _into your conda environment_ using the following command
+   ```
+   pip install -r requirements.txt
+   ```
+   where the `requirements.txt` file contains a list of python libraries
+9. And now build the book:
+   ```
+   jupyter-book build .
+   ```
+10. The output of the build process will provide output such as below that demonstrate how you can view the book locally:
+    ```
+    ===============================================================================
+
+    Finished generating HTML for book.
+    Your book's HTML pages are here:
+        _build/html/
+    You can look at your book by opening this file in a browser:
+        _build/html/index.html
+    Or paste this line directly into your browser bar:
+        file:///[...]/alan-turing-institute/the-turing-way/book/website/_build/html/index.html
+
+    ===============================================================================
+    ```
+    
+### Build the book while working on a Pull Request
+If you would like to preview a version of the book from a certain branch (perhaps to render the book while working on a PR) then simply switch to the required branch and rebuild the book as in step 9:
+   ```
+   git checkout mybranch
+   jupyter-book build .
+   ```
+Follow the link as before and you will see changes specific to that branch rendered.
+
+## Why did we recommend using (mini)conda?
+In the step-by-step guide above, we made use of the `jupyter-book` command to build the Turing Way book. For this command to work as intended you will need a python installation on your machine. 
+As with any other programming language such as R or Julia, any given python installation might look different from another due to the different packages or libraries that come with the installation.
+Over time you will likely install even more packages, or update packages to newer versions. Some packages also depend on the presence of specific versions of other packages to function, and so to ensure your local build works smoothly you will want to minimize as much mismatched dependencies as possible.
+
+But this can be difficult! Even with an organized, concerted effort, package management for programming languages naturally throws up dependency issues. Python packages, for reasons not discussed here, tend to suffer from dependency issues a bit more than other languages (note that all languages do!) and one guaranteed way to come across such an issue by trying to maintain all of your python projects using just one, large set of packages, each at a specific version. You simply cant cater to the needs of all package dependencies this way. https://xkcd.com/1987/   
```

Comment:
```suggestion
But this can be difficult! Even with an organized, concerted effort, package management for programming languages naturally throws up dependency issues. Python packages, for reasons not discussed here, tend to suffer from dependency issues a bit more than other languages (note that all languages do!) and one guaranteed way to come across such an issue by trying to maintain all of your python projects using just one, large set of packages, each at a specific version. You simply can't cater to the needs of all package dependencies this way. https://xkcd.com/1987/   
```

missing apostrophe